packages/k8s/src/hooks/prepare-job.ts

```diff
@@ -125,12 +125,11 @@ function generateResponseFile(
   )
   if (serviceContainers?.length) {
     response.context['services'] = serviceContainers.map(c => {
-      if (!c.ports?.length) {
-        return { image: c.image }
-      }
       const ctxPorts: ContextPorts = {}
-      for (const port of c.ports) {
-        ctxPorts[port.containerPort] = port.hostPort
+      if (c.ports) {
```

Comment:
```suggestion
      if (c.ports?.length) {
```
NIT: we don't need to try iterating if the length is 0